vitrina/orgs/models.py

```diff
@@ -167,6 +167,10 @@ class Representative(models.Model):
     content_object = GenericForeignKey("content_type", "object_id")
     can_write = models.BooleanField(default=False, verbose_name=_("Leidžiama keisti duomenis"))
     can_make_agreements = models.BooleanField(default=False, blank=True, null=True)
+    information_system_representative = models.BooleanField(
+        default=False, verbose_name=_("Informacinės sistemos tvarkytojas")
+    )
+    open_data_representative = models.BooleanField(default=False, verbose_name=_("Atvirų duomenų tvarkytojas"))
```

Comment:
According to naming practices:
```suggestion
    is_information_system_representative = models.BooleanField(
        default=False, verbose_name=_("Informacinės sistemos tvarkytojas")
    )
    is_open_data_representative = models.BooleanField(default=False, verbose_name=_("Atvirų duomenų tvarkytojas"))
```